ledger-api/README.md

```diff
@@ -38,27 +34,24 @@ sbt test
 
 # Release
 
-The release process on jenkins also publishes the [Ledger API Introduction](docs/ledger-api-introduction) documentation package with the release version.
-
 1. Create a branch `ledger-api-release-<major>-<minor>-<point>` for the release. Check the file [ledger-api/version.sbt](version.sbt). For example, let's say the the current
 version of the project is `0.9.3-SNAPSHOT`. You need to create a branch called `ledger-api-release-0-9-3`;
-2. Generate the protobuf ProtoDocs with `ledger-api/docs/ledger-api-introduction/generate-protodocs` and commit to that branch.
-3. Bump the version for the NPM distribution channel:
+2. Bump the version for the NPM distribution channel:
 
       (cd npm-package && npm version <major>.<minor>.<point>)
 
-4. Move the ## Unreleased section of [ledger-api/UNRELEASED.md](UNRELEASED.md) to the new release
+3. Move the ## Unreleased section of [ledger-api/UNRELEASED.md](UNRELEASED.md) to the new release
 version you have created in [ledger-api/CHANGELOG.md](CHANGELOG.md) and commit to that branch. The message of the commit is not important as it
 will be discarded;
-5. Run `sbt release`. This will ask for the release version and the next version, and will create
+4. Run `sbt release`. This will ask for the release version and the next version, and will create
 commit with the release version and next version, and also takes care about tagging;
-6. Push your **branch and tag**:
+5. Push your **branch and tag**:
 
 ```
 git push origin release/ledger-api/0.9.3        # check the tag that has been created, and push it
 git push -u origin ledger-api-release-0-9-3    # push your branch
 ```
-7. Go to the release [Jenkins job](http://ci.da-int.net/job/ledger-api/job/release/build?delay=0sec)
+6. Go to the release [Jenkins job](http://ci.da-int.net/job/ledger-api/job/release/build?delay=0sec)
```

Comment:
This is outdated too. We should probably link to our release procedure instead or just omit the release section in general.

Reply:
I thought so. Where is the release procedure? If you give me a link I'll remove this section and sub it in instead.

Reply:
I am not sure where it is documented currently, but as we are doing head based development  the best would be I think is just removing the release section from here.

Reply:
Done!